src/cli.ts

```diff
@@ -1,55 +0,0 @@
-#! /usr/bin/env node
-import * as core from '@actions/core';
-import chalk from 'chalk';
-import updateNotifier from 'update-notifier';
-
-import pkg from '../package.json' with { type: 'json' };
-
-import { isGHA } from './lib/isCI.js';
-
-import rdme from './index.js';
-
-updateNotifier({ pkg }).notify();
-
-rdme(process.argv.slice(2))
-  .then((msg: string) => {
-    if (msg) {
-      // eslint-disable-next-line no-console
-      console.log(msg);
-      if (isGHA()) {
-        core.setOutput('rdme', msg);
-      }
-    }
-    return process.exit(0);
-  })
-  .catch((err: Error) => {
-    let message = `Yikes, something went wrong! Please try again and if the problem persists, get in touch with our support team at ${chalk.underline(
-      'support@readme.io',
-    )}.`;
-
-    if (err.message) {
-      message = err.message;
-    }
-
-    /**
-     * If we're in a GitHub Actions environment, log errors with that formatting instead.
-     *
-     * @see {@link https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message}
-     * @see {@link https://github.com/actions/toolkit/tree/main/packages/core#annotations}
-     */
-    if (isGHA()) {
-      return core.setFailed(message);
-    }
```

Comment:
Will probably want to retain this somewhere.

Reply:
it's been a bit since i wrote all the `baseCommand` logic but yeah i can look into it. tbh i don't actually think this function is doing much but i can double check to see if the error output is any different:

https://github.com/actions/toolkit/blob/bb2278e5cfbb40afc20890c415e9ffa836631cd5/packages/core/src/core.ts#L220-L224